### PR TITLE
DE39499 Cert 20.20.7: Translation updates

### PR DESCRIPTION
**Pull request by emil-furniss**
Cert for #953

### CODE REVIEW

File: components/d2l-activity-editor/lang/ar.js [hunk 1]
```diff
@@ -55,9 +55,9 @@ export default {
 	"noGradeCreatePermission": "ليس لديك الإذن لإنشاء مادة جديدة سيتم وضع علامة عليها", // Reason why a new grade items cannot be created in the choose grades dialog
 	"competencies": "أهداف التعلّم", //Text label for the competencies tool integration
 	"manageCompetencies": "إدارة أهداف التعلّم", //Button text to launch competencies tool dialog
-	"competenciesCount": "{count, plural, =0 {No learning objectives} =1 {1 attached} other {{count} attached}}", //Label for number of associated competencies
+	"competenciesCount": "{count, plural, =0 {ما من أهداف تعلّم} =1 {مرفق واحد} other {{count} من المرفقات}}", //Label for number of associated competencies
```
Comment (wongvincent) [inline components/d2l-activity-editor/lang/ar.js:58]: is the 1 flipped? should it be `=1 {مرفق واحد}`

File: components/d2l-activity-editor/lang/de.js [hunk 2]
```diff
@@ -55,9 +55,9 @@ export default {
 	"noGradeCreatePermission": "Sie haben keine Berechtigung, einen neuen Benotungsgegenstand zu erstellen.", // Reason why a new grade items cannot be created in the choose grades dialog
 	"competencies": "Lernziele", //Text label for the competencies tool integration
 	"manageCompetencies": "Lernziele verwalten", //Button text to launch competencies tool dialog
-	"competenciesCount": "{count, plural, =0 {No learning objectives} =1 {1 attached} other {{count} attached}}", //Label for number of associated competencies
+	"competenciesCount": "{count, plural, =0 {Keine Lernziele} =1 {1 angehängt} andere {{count} angehängt}}", //Label for number of associated competencies
```
Comment (wongvincent) [inline components/d2l-activity-editor/lang/de.js:58]: they translated `other` :(

File: components/d2l-activity-editor/lang/de.js [hunk 3]
```diff
@@ -55,9 +55,9 @@ export default {
...
 	"competenciesCountSummary": "{count, plural, =0 {Keine Lernziele} =1 {1 Lernziel} andere {{count} Lernziele}}",
-	"unevaluatedCompetencies": "{count, plural, =1 {1 not being evaluated} other {{count} not being evaluated}}", //Label for number of unevalated associated competencies
+	"unevaluatedCompetencies": "{count, plural, =1 {1 fehlende Bewertung} andere {{count} fehlende Bewertungen}}", //Label for number of unevalated associated competencies
```
Comment (wongvincent) [inline components/d2l-activity-editor/lang/de.js:60]: translated `other` :(

File: components/d2l-activity-editor/lang/fr.js [hunk 4]
```diff
@@ -55,9 +55,9 @@ export default {
 	"noGradeCreatePermission": "Vous n\'êtes pas autorisé(e) à créer un élément de note", // Reason why a new grade items cannot be created in the choose grades dialog
 	"competencies": "Objectifs d\'apprentissage", //Text label for the competencies tool integration
 	"manageCompetencies": "Gérer les objectifs d’apprentissage", //Button text to launch competencies tool dialog
-	"competenciesCount": "{count, plural, =0 {No learning objectives} =1 {1 attached} other {{count} attached}}", //Label for number of associated competencies
+	"competenciesCount": "{count, plural, =0 {Aucun objectif d’apprentissage} =1 {1 attached} other {{count} attached}}", //Label for number of associated competencies
```
Comment (wongvincent) [inline components/d2l-activity-editor/lang/fr.js:58]: didn't translate 1 or other I think

Comment (emil-furniss) [reply components/d2l-activity-editor/lang/fr.js:58]: I think `attached` actually is the french word as well

File: components/d2l-activity-editor/lang/ar.js (hunk 1 above)
Comment (emil-furniss) [reply components/d2l-activity-editor/lang/ar.js:58]: This is actually correct, just the rtl + ltr in one line messing with the rendering here. Shows up just fine in FACE.

**Pull request merged**